### DISCUSSION TITLE
enable-auto-groups &  auto-group-groupname

### DISCUSSION THREAD

**Discussion by KatelynnChan** [RoinujNosde/SimpleClans#155] [Q&A] [answered]
What does  enable-auto-groups do when enabled or disabled and what does  auto-group-groupname when enabled or disabled?

**Comment by RoinujNosde** [comment, marked as answer]
enable-auto-groups adds players into groups related to their position in their clan: sc_trusted, sc_leader, sc_CLANTAG

auto-group-groupname gives players the permission `group.CLANTAG`, which in LuckPerms is the same as adding the player to the group `CLANTAG`.
